packages/react-form/src/hooks/field/test/field.test.tsx

```diff
@@ -14,7 +14,7 @@ describe('useField', () => {
           <input
             id="test-field"
             name="test-field"
-            value={field.value}
+            value={field.value as any}
```

Comment:
Bummer, why was this needed?

Reply:
Because in one of the tests we use an object as the value and the HTML input haaaates it.  Maybe I'll just go back and make a different test component for that.